src/central-submissions.rst

```diff
@@ -143,3 +147,52 @@ To connect with Excel or Power BI, follow these steps.
 
 If you want to use the free and popular `R statistics and analysis tool <https://www.r-project.org/>`_, we recommend you use `ruODK <https://docs.ropensci.org/ruODK/>`_. A guide for getting started with it can be found `here <https://docs.ropensci.org/ruODK/articles/odata-api.html>`_. ruODK is developed and supported by community members. If you wish to help improve it, you can find information `on GitHub <https://docs.ropensci.org/ruODK/CONTRIBUTING.html>`_.
 
+.. tip::
+  If you are having trouble getting Power BI to connect to Central, and especially if you see error messages about permissions or authentication, try `resetting the Power BI cached credentials <https://community.powerbi.com/t5/Power-Query/Power-BI-Web-cached-credentials-For-OData/m-p/126826/highlight/true#M8228>`_.
+
+You can also access the OData feed yourself. The OData feed is an easily consumable JSON data format and offers a metadata schema, some filtering and paging options, and more. To learn more about the OData feed, click the :guilabel:`API Access` button or see the `developer documentation <https://odkcentral.docs.apiary.io/#reference/odata-endpoints>`_ directly.
+
+.. _central-submissions-review-states:
+
+Submission Review States
+------------------------
+
+As of version 1.2, Central allows Project Managers and Administrators to review submissions and assign them certain states. This optional feature lets you perform verification and data correction within Central itself. The available states are:
```

Comment:
s/optional//
s/correction/editing//

Reply:
hmm. i feel like it's useful to soften the feature since we sort of force the "received" state on everyone to indicate that it's not necessary to do anything w review states?

Reply:
Editing feels more consistent and easier to understand to me.

"This optional feature" makes me want to look for a way to opt in/out. Maybe "...to review submissions and assign them certain states. Submissions can also be edited from Central." If you want to emphasize that review states can be entirely ignored, maybe the language from https://github.com/getodk/docs/pull/1346/files/68d3da317c66f732ba08537cfbd9cd3443777f02..552fea1b4bb7394463e3fada4917a4ef186ffa30#r624405747 could be moved up.

I think of editing and review states as two separate things. Maybe "Submission Review States" could be "Submission Editing and Review States"?

Reply:
i didn't think this section was about editing at all. i describe editing  in Submission Details. it probably deserves its own section.

Reply:
I like what you came up with here. 👍